addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -93,29 +118,27 @@ $i: 1;
         }
     }
 
-    we-button.o_we_mobile {
+    we-button.o_we_device {
         > div {
             // FIXME maybe there is some refactoring to do so that all SVG icons
             // can be properly aligned with extra CSS rules...
             display: flex;
             align-items: center;
 
             svg {
-                width: 15px;
-                fill: $o-we-color-danger;
+                width: 12px;
+                fill: $o-we-sidebar-content-field-clickable-color;
                 margin-bottom: 0;
-                margin-left: -3px;
 
                 &:hover {
-                    fill: darken($o-we-color-danger, 7.5%);
+                    fill: $o-we-sidebar-content-field-pressed-color;
```

Comment:
Is not this the default rules? Can't this be removed?

Reply:
The `fill` property defaults to `rgb(0, 0, 0)`. Since we want the icon to be lighter than the background of the button we need to specify `fill` - and to get some visual feedback we need to specify it for `:hover` as well.